hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/resources/mapred-default.xml

```diff
@@ -286,6 +286,13 @@
   </description>
 </property>
 
+<property>
+  <name>mapreduce.task.enable.ping-for-liveliness-check</name>
+  <value>true</value>
```

Comment:
The value must be same with the DEFAULT_MR_TASK_ENABLE_PING_FOR_LIVELINESS_CHECK.

Reply:
Thanks @iwasakims for review. I have updated the DEFAULT_MR_TASK_ENABLE_PING_FOR_LIVELINESS_CHECK to true.

Reply:
When should we turn this off? If it should be always true, we don't need to make it configurable. We usually needs new configuration knob to add new feature disabled by default for compatibility and safety.

Reply:
Yeah I think this makes sense, I will remove configurable part.

Reply:
Hi @iwasakims. I saw the comment from @cnauroth on JIRA about his suggestion to put it behind the configuration and I think we can keep it configurable and let the end user decide on it. By default, lets keep it 

```mapreduce.task.enable.ping-for-liveliness-check : false``` which will keep the current behaviour intact

and for use cases that we saw in JIRA, user can set it true when required

Thoughts?/

Reply:
@ashutoshcipher Making this feature disabled by default sounds right direction as @cnauroth mentioned. While no unexpected side effect was observed in the test results of current patch, there should be additional test case covering the code path of new feature enabled.

Reply:
Thanks @iamcaoxudong, let me make the required changes.

Reply:
@iwasakims @cnauroth - I have made the required changes, Can you please review the PR? Thanks